exporter/chronicleexporter/README.md

```diff
@@ -14,57 +14,39 @@ This exporter facilitates the sending of logs to Chronicle, which is a security
+| `customer_id`       | string |                                                | `false`  | The customer ID used for sending logs.                                                      |
+| `override_log_type` | bool   | `true`                                         | `false`  | Whether or not to override the `log_type` in the config with `attributes["log_type"]`       |
+| `namespace`         | string |                                                | `false`  | User-configured environment namespace to identify the data domain the logs originated from. |
 
 ### Log Type
 
 If the `attributes["log_type"]` field is present in the log, and maps to a known Chronicle `log_type` the exporter will use the value of that field as the log type. If the `attributes["log_type"]` field is not present, the exporter will use the value of the `log_type` configuration field as the log type.
 
 ## Credentials
 
-This exporter requires a Google Cloud service account with access to the Chronicle API. The service account must have access to the following endpoint(s):
-
-The base endpoint is `https://malachiteingestion-pa.googleapis.com`
-
-Alternatively, if a `region` is specified:
-
-| Region                       | Endpoint                                                            |
-| ---------------------------- | ------------------------------------------------------------------- |
-| `Europe Multi-Region`        | `https://malachiteingestion-pa-europe.googleapis.com`               |
-| `Frankfurt`                  | `https://malachiteingestion-pa-europe-west3.googleapis.com`         |
-| `London`                     | `https://malachiteingestion-pa-europe-west2.googleapis.com`         |
-| `Singapore`                  | `https://malachiteingestion-pa-asia-southeast1.googleapis.com`      |
-| `Sydney`                     | `https://malachiteingestion-pa-australia-southeast1.googleapis.com` |
-| `Tel Aviv`                   | `https://malachiteingestion-pa-europe-west4.googleapis.com`         |
-| `United States Multi-Region` | `https://malachiteingestion-pa.googleapis.com`                      |
-| `Zurich`                     | `https://malachiteingestion-pa-europe-west6.googleapis.com`         |
+This exporter requires a Google Cloud service account with access to the Chronicle API. The service account must have access to the endpoint specfied in the config.
+Besides, the default endpoint, there are also regional endpoints that can be used [here](https://cloud.google.com/chronicle/docs/reference/ingestion-api#regional_endpoints)
+.
 
 For additional information on accessing Chronicle, see the [Chronicle documentation](https://cloud.google.com/chronicle/docs/reference/ingestion-api#getting_api_authentication_credentials).
 
 ## Example Configuration
 
 ```yaml
 chronicle:
-  region: "Europe Multi-Region"
```

Comment:
Do we need an endpoint field in this example?

Reply:
I figured no because we have a default endpoint that'd get populated

Reply:
Can we should an example with an endpoint?